src/commonMain/kotlin/exchange.dydx.abacus/state/manager/SystemUtils.kt

```diff
@@ -0,0 +1,12 @@
+package exchange.dydx.abacus.state.manager
+
+enum class Platform(val rawValue: String) {
+    ios("ios"),
+    android("android"),
+    web("web"),
+}
+expect class SystemUtils {
```

Comment:
`expect` here is a way for Kotlin Multiplat to define a protocol which expects an `actual` implementation for each supported platform